tests/test_i2s_callback_sequence/Makefile

```diff
@@ -75,10 +83,6 @@ else
   endif
 endif
 
-# The XCORE_ARM_PROJECT variable, if set to 1, configures this
-# project to create both xCORE and ARM binaries.
-XCORE_ARM_PROJECT = 0
```

Comment:
It may be worth checking with @xross to see if he knows why we ever tried to build I2S for Arm.  I vaguely remember some comments about a package that had both an XCORE and an Arm processor.  I have no idea if we still sell it or not.

Reply:
This is a legacy line from when the tools could target the XCORE XA device. It will be ignored these days so the line can be removed.
We do still sell it I believe, but it is certainly not for design in.